projects/plugins/jetpack/extensions/blocks/slideshow/controls.js

```diff
@@ -38,77 +35,76 @@ export default ( {
 
 	return (
 		<Fragment>
-			<InspectorControls>
-				<PanelBody title={ __( 'Autoplay', 'jetpack' ) }>
-					<ToggleControl
-						label={ __( 'Autoplay', 'jetpack' ) }
-						help={ __( 'Autoplay between slides', 'jetpack' ) }
-						checked={ autoplay }
+			<PanelBody title={ __( 'Autoplay', 'jetpack' ) }>
+				<ToggleControl
+					label={ __( 'Autoplay', 'jetpack' ) }
+					help={ __( 'Autoplay between slides', 'jetpack' ) }
+					checked={ autoplay }
+					onChange={ value => {
+						setAttributes( { autoplay: value } );
+					} }
+				/>
+				{ autoplay && (
+					<RangeControl
+						label={ __( 'Delay between transitions (in seconds)', 'jetpack' ) }
+						value={ delay }
 						onChange={ value => {
-							setAttributes( { autoplay: value } );
+							setAttributes( { delay: value } );
 						} }
+						min={ 1 }
+						max={ 5 }
 					/>
-					{ autoplay && (
-						<RangeControl
-							label={ __( 'Delay between transitions (in seconds)', 'jetpack' ) }
-							value={ delay }
-							onChange={ value => {
-								setAttributes( { delay: value } );
-							} }
-							min={ 1 }
-							max={ 5 }
-						/>
-					) }
-					{ autoplay && prefersReducedMotion && (
-						<span>
-							{ __(
-								'The Reduce Motion accessibility option is selected, therefore autoplay will be disabled in this browser.',
-								'jetpack'
-							) }
-						</span>
-					) }
-				</PanelBody>
-				<PanelBody title={ __( 'Effects', 'jetpack' ) }>
+				) }
+				{ autoplay && prefersReducedMotion && (
+					<span>
+						{ __(
+							'The Reduce Motion accessibility option is selected, therefore autoplay will be disabled in this browser.',
+							'jetpack'
+						) }
+					</span>
+				) }
+			</PanelBody>
+			<PanelBody title={ __( 'Effects', 'jetpack' ) }>
+				<SelectControl
+					label={ __( 'Transition effect', 'jetpack' ) }
+					value={ effect }
+					onChange={ value => {
+						setAttributes( { effect: value } );
+					} }
+					options={ effectOptions }
+				/>
+			</PanelBody>
+			{ ! isEmpty( images ) && ! isEmpty( imageSizeOptions ) && (
+				<PanelBody title={ __( 'Image Settings', 'jetpack' ) }>
 					<SelectControl
-						label={ __( 'Transition effect', 'jetpack' ) }
-						value={ effect }
-						onChange={ value => {
-							setAttributes( { effect: value } );
-						} }
-						options={ effectOptions }
+						label={ __( 'Image Size', 'jetpack' ) }
+						value={ sizeSlug }
+						options={ imageSizeOptions }
+						onChange={ onChangeImageSize }
 					/>
 				</PanelBody>
-				{ ! isEmpty( images ) && ! isEmpty( imageSizeOptions ) && (
-					<PanelBody title={ __( 'Image Settings', 'jetpack' ) }>
-						<SelectControl
-							label={ __( 'Image Size', 'jetpack' ) }
-							value={ sizeSlug }
-							options={ imageSizeOptions }
-							onChange={ onChangeImageSize }
-						/>
-					</PanelBody>
-				) }
-			</InspectorControls>
-			<BlockControls>
-				{ !! images.length && (
-					<ToolbarGroup>
-						<ToolbarItem>
```

Comment:
I see you've removed `ToolbarItem`. I added it in #18860 to address these changes:
https://make.wordpress.org/core/2020/11/18/changes-to-toolbar-components-in-wordpress-5-6/

Do you think we should get rid of it, or can we keep it here?

Reply:
My mistake here. When running through Jest `ToolbarItem` was erroring as undefined and I read some out-of-date docs which seemed to indicate it isn't needed. You are correct though that it should be there.

I have added it back and the reason it shows as undefined in jest tests is because the `@wordpress/components` dependency in the jetpack project is `9.2.6` which still as `ToolbarItem` as `__experimentalToolbarItem`.

I assume that the build process is pulling in a newer dependency somehow via the webpack `'@automattic/calypso-build/webpack.config.js'. I tried forcing a newer version of  `@wordpress/components`  in `package.json` but this made things more broken. @brbrr, do you have any ideas on how to get jest using the same dependency tree for the `@wordpress` dependencies as the webpack builds?

Reply:
Ah right, I ran into the same issue in #18860, but had forgotten about it :/

I had tried to update our dependencies, but was running into other issues; I ended up reverting:
https://github.com/Automattic/jetpack/pull/18860/commits/b91bb2a6dcca076383beecf8def8eb093508b784

We'll need to merge #16763 before we can do anything. Until then, maybe we can remove `ToolbarItem` like you had done, with a note like the one I left in https://github.com/Automattic/jetpack/pull/18860/commits/b91bb2a6dcca076383beecf8def8eb093508b784 ?

Unless @brbrr has a better idea :)

Reply:
At this point I have just removed the single test for the toolbar, which was only doing a very basic 'did it render' sort of test, so not critical, and have added a comment in the spec file to indicate why there is no test for this component.